crates/context_aware_config/src/api/config/handlers.rs

```diff
@@ -595,32 +599,23 @@ async fn get_resolved_config(
 
     config = apply_prefix_filter_to_config(&mut query_params_map, config)?;
 
-    let cac_client_contexts = config
-        .contexts
-        .into_iter()
-        .map(|val| cac_client::Context {
-            condition: json!(val.condition),
-            override_with_keys: val.override_with_keys,
-        })
-        .collect::<Vec<_>>();
```

Comment:
INFO: was earlier needed due to type disparity between server side `Config` type and client side `Config` type